x-pack/solutions/security/packages/kbn-securitysolution-autocomplete/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@kbn/securitysolution-autocomplete",
   "version": "1.0.0",
   "description": "Security Solution auto complete",
-  "license": "Elastic License 2.0 OR AGPL-3.0-only OR SSPL-1.0",
+  "license": "Elastic License 2.0",
```

Comment:
@gsoldevila moving this package under x-pack changes it's licence. Should this maybe go under src/solutions/security/packages ?

Reply:
nvm, read through the RFC / technical design again and this licence change should not be a problem.